draft-packages/avatar/KaizenDraft/Avatar/decision-tokens.scss

```diff
@@ -5,3 +5,10 @@ $dt-color-background-color-personal: $color-orange-100;
 $dt-color-text-color: $color-purple-800;
 $dt-color-border-color: $color-white;
 $dt-color-icon-color: rgba($color-purple-800-rgb, 0.7);
+
+// Proposed Values
+$dt-avatar-sm: 1.5rem;
```

Comment:
Don't worry about this now that you've done it—but FYI we're wanting to move away from these `decision-tokens` files. So if you were to do this again, I'd recommend that you just create these variables from within the `styles.module.scss` file.